src/bp/migrations/v12_22_1-1621260400-add_file_and_location_content_type_bot_config.ts

```diff
@@ -0,0 +1,53 @@
+import * as sdk from 'botpress/sdk'
+import { Migration, MigrationOpts } from 'core/migration'
+import _ from 'lodash'
+
+const migration: Migration = {
+  info: {
+    description: 'Adds file and location content types to bot configs',
+    target: 'core',
+    type: 'config'
+  },
```

Comment:
Bot migrations are a bit different from core migrations. Don't forget that there are two scenarios where a bot can be migrated:
1. When the whole botpress server is migrated to another version
2. When you import a bot

The signature is different to handle both cases. The target must be `bot` and there is a new property called `metadata` which has the ID of the bot (if it's undefined, then we process all bots)

Also, I just realized that the previous migration for video/audio doesn't even implement this correctly, I fixed it here: https://github.com/botpress/botpress/pull/5049

Reply:
Right! Thanks a lot for the clarification!!